spec/quality_spec.rb

```diff
@@ -191,6 +191,58 @@ def check_for_specific_pronouns(filename)
     expect(error_messages.compact).to be_well_formed
   end
 
+  it "documents all used settings" do
+    exemptions = %w(
```

Comment:
I am okay with none of them being documented right now. I would like to document all of them eventually, so we should probably create a meta-ticket that lists all of them so that we can track getting them documented.